projects/web-app/src/main.js

```diff
@@ -2,11 +2,11 @@ import { createApp } from 'vue';
 import '@/sw/register';
 import router from '@/router';
 // import store from '@/store';
-import App from '@/App.vue';
+import AppShell from '@/components/app-shell/app-shell';
```

Comment:
@vvandijck I know you got rid of my 'folder-with-multiple-components' structure; now it's back because I feel it's pretty clean to group certain component in one folder - especially when following the new naming convention.